specs/src/parameters_v4.md

```diff
@@ -0,0 +1,77 @@
+# Parameters v4
+
+The parameters below represent the parameters for app version 4.
+
+Note that not all of these parameters are changeable via governance. This list
```

Comment:
Should we mention that params are no longer changed via v1beta1 `ParamChangeProposal` but with a `MsgSubmitProposal` which includes a msg of `moduleXYZ.MsgUpdateParams`

Reply:
cb4e813534efcb4026b90a5f2c30029a81547ff8